lib/data_magic/error_checker.rb

```diff
@@ -99,6 +108,8 @@ def build_error(opts)
             "The range '#{opts[:input]}' supplied to parameter '#{opts[:parameter]}' isn't in the correct format."
           when 'zipcode_error'
             "The provided zipcode, '#{opts[:input]}', is not valid."
+          when 'distance_error'
+            "Distance requires zipcode (none given)."
```

Comment:
How about `Use of the distance parameter also requires a zip parameter.` (I like more readable errors.)